From eea89adf0c38754d2a985baefb4ac85e8eb72183 Mon Sep 17 00:00:00 2001
From: Alex Naidis <alex.naidis@linux.com>
Date: Thu, 13 Oct 2016 18:53:44 +0200
Subject: [PATCH 13/19] build: ensure QCLD usage when requested

* Passing -fuse-ld=qcld to cflags
  is required to make sure compiler
  chooses QCLD as linker.

Change-Id: Ifa7b5e40c51476e14a3b50465e2f5b58abeb8fee
Signed-off-by: Alex Naidis <alex.naidis@linux.com>
Signed-off-by: Joe Maples <joe@frap129.org>
---
 core/binary.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/binary.mk b/core/binary.mk
index b03d8579f..63413484c 100644
--- a/core/binary.mk
+++ b/core/binary.mk
@@ -550,7 +550,7 @@ my_target_global_ldflags := $($(LOCAL_2ND_ARCH_VAR_PREFIX)CLANG_$(my_prefix)GLOB
 
         ifeq ($(LOCAL_SDCLANG_LTO), true)
         ifneq ($(LOCAL_MODULE_CLASS), STATIC_LIBRARIES)
-            SDCLANG_PRECONFIGURED_FLAGS += -flto
+            SDCLANG_PRECONFIGURED_FLAGS += -fuse-ld=qcld -flto
             my_target_global_ldflags += -fuse-ld=qcld -flto
         endif
         endif
-- 
2.19.1

